nservicebus/operations/metrics.md

```diff
@@ -10,61 +10,76 @@ related:
 ---
 
 The `NServiceBus.Metrics` package can be used to collect information and measure endpoint health and performance. When a system is broken down into multiple processes, each with its own queue, then bottlenecks may be identified by tracking:
- - the number of messages in each queue
- - the incoming rate of messages for each queue
- - the processing rate for each queue.
+
+ * the number of messages in each queue
+ * the incoming rate of messages for each queue
+ * the processing rate for each queue.
+
 
 ## Enabling NServiceBus.Metrics
 
 snippet: Metrics-Enable
 
 partial: reporting
 
+
 ## Metrics captured
 
-NServiceBus and ServiceControl capture a number of different metrics about a running endpoint including the processing time, "critical time", and number of messages in each queue.
+NServiceBus and ServiceControl capture a number of different metrics about a running endpoint including the processing time, the number of messages in each queue (differentiating between those pulled from the queue, those processed successfully, and those which failed processing), as well as "critical time".
```

Comment:
Should we be mentioning NServiceBus and ServiceControl or should we just focus on the Metrics plugin?